### ISSUE TITLE
Improved navigation and search in the conversation (expand and scroll to quoted message, scroll to search results, etc.).

### ISSUE THREAD

**Issue by epoberezkin** [simplex-chat/simplex-chat#2314]

**Comment by Dituar** [comment]
As for improving the search.. so that when searching for “word”, “Word” is found, and also like in Russian, the search for “зеленый” finds “зелёный”, because keyboard autocorrection often replaces words with "ё".

**Comment by justanediblefriend** [comment]
Seconding (fourthing?) this issue, so unfortunate that so many important messages get lost due to the impossibility of scrolling back to the context they're in. Really hope this gets added soon, appreciate the labor going into this fantastic app.

**Comment by CKMz17** [comment]
Yes, it's true that it is very inconvenient not being able to return to the context of the message. In large groups, it's immediately noticeable. I hope you can implement it as soon as possible, thank you
